### PR TITLE
Fix links in documentation

### PR DESCRIPTION
**Pull request by LaiWang2020**
Fixes #{3885}

Changes proposed in this pull request:
Hi, I also encountered this problem and try to solve the problem according to my understanding. However, as I read the comments, I don't know if this problem is closed now. If it is closed, I'm sorry to waste your time to review and remind you that this problem should be closed.

### CODE REVIEW

File: extensions/database/src/com/google/refine/extension/database/DatabaseConfiguration.java [hunk 1]
```diff
@@ -42,8 +42,8 @@
     
     //optional parameters
     private boolean useSSL;
-    
-    
+
+
```
Comment (wetneb) [inline extensions/database/src/com/google/refine/extension/database/DatabaseConfiguration.java:46]: Could you perhaps cancel this spurious and unrelated change?

**Review by wetneb — APPROVED** [review]
Thank you very much!

**Comment by wetneb** [comment]
This replaces #3885

File: extensions/database/src/com/google/refine/extension/database/DatabaseConfiguration.java (hunk 1 above)
Comment (LaiWang2020) [reply extensions/database/src/com/google/refine/extension/database/DatabaseConfiguration.java:46]: Sure, sorry for that.
